src/Markdig.Wpf.SampleApp/MainWindow.xaml.cs

```diff
@@ -1,7 +1,9 @@
 ﻿using System.Diagnostics;
-using Markdig.Renderers;
+using System.IO;
 using System.Windows;
 using System.Windows.Documents;
+using Markdig.Renderers;
```

Comment:
This `using` is not needed anymore since we will remove code from the constructor (as well as the previous one).